src/views/v2/emails/template-parts/body/order/purchaser-details/date.php

```diff
@@ -23,9 +23,10 @@
  * @var \WP_Post                           $order              The order object.
  */
 
-$order_date = get_the_date( get_option( 'date_format' ), $order );
+use Tribe__Date_Utils as Dates;
```

Comment:
I might be wrong, but I find the usage of classes within templates to build data on the fly is something we try to avoid, right?

Reply:
We should, but in this case trying to remove it will take more global changes that are not worth the time now.